templates/licenses/MIT/license

```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) <%= params.creationDate %> <%= params.author %>
```

Comment:
why do we have different pattern for templates with <% params.creationDate %> vs {date} 

?

Reply:
Because license header varies. We should check if license is supported then select correct template. Sometimes we do not need it at all (in case of no license or invalid id).
If you know better solution, please share :)